content/en/docs/Reference/auroraboot.md

```diff
@@ -599,62 +603,58 @@ This method differs from the ones documented in the [Build raw images with QEMU]
 Consider the following example:
 
 ```bash
-docker run -v /var/run/docker.sock:/var/run/docker.sock --net host \
+docker run -P -v /var/run/docker.sock:/var/run/docker.sock \
   -v $PWD:/aurora --rm -ti quay.io/kairos/auroraboot \
   --debug \
   --set "disable_http_server=true" \
-  --set "container_image={{<oci variant="standard">}}" \
   --set "disable_netboot=true" \
-  --cloud-config /aurora/config.yaml \
-  --set "disk.raw=true" \
+  --set "disk.efi=true" \
+  --set "container_image={{<oci variant="standard">}}" \
   --set "state_dir=/aurora"
 ```
 
-The raw disk image will be available in `iso/disk.raw`
+The raw disk image will be available in the current directory with the `.raw` image extension
+
+To generate a BIOS raw image just change `disk.efi=true` for `disk.bios=true`
 
-The cloud config file (`config.yaml`) should look like the following:
+The cloud config file (`01_reset.yaml`) in the raw image oem partition should look like the following:
```

Comment:
Is this manual step really needed? Don't we do this by default?

Reply:
yes, but its more of an informative thing rigth? before we mentioned it so maybe now we also need to do so? no idea, maybe this needs a rework